styles.css

```diff
@@ -150,17 +150,21 @@
     gap: 1em;
     flex-direction: column;
     align-items: center;
-    margin-top: -4em;
+    margin-top: -10em;
   }
 
   .menu-item {
-    width: 60%;
-    font-size: 1.5rem;
+    width: 90%;
+    font-size: 1.7rem;
     font-weight: 700;
     line-height: 2rem;
     color: var(--color-lightsalmon);
     padding: 0 0 0.3em 0.5em;
-    border-bottom: 1px solid var(--color-whitesmoke);
+    border-bottom: 1px solid #f4f1f02b;
+  }
+
+  .menu-item:nth-last-child(1) {
+    display: none;
```

Comment:
- [ ] Great job so far, :+1: :+1: You did great implementing some of the changes pointed out by the previous reviewer, But I think there are some improvements you need to make to your design in order to match the given design :wave: 
- - As pointed out by the previous reviewer, and I have also marked it on the screenshot below, your menu list items are meant to be at the top and not at the center, I'll suggest you update your styling to match the Figma menu design as part of the requirement fo this milestone is to replicate the given design as much as you can. :smile: :wave: 

- Your current design Screenshot:
![Selection_042](https://github.com/lily4178993/Portfolio/assets/107632780/7c4cad4d-7495-4091-91fa-6f26392a89a2)

- Figma
![Screenshot from 2023-05-17 01-19-46](https://github.com/lily4178993/Portfolio/assets/107632780/22b10d6a-6850-4c7d-b4f3-f6620c211aac)
 design: